saleor/graphql/page/tests/test_page.py

```diff
@@ -127,19 +127,18 @@ def test_staff_query_unpublished_page(staff_api_client, page, permission_manage_
 
 CREATE_PAGE_MUTATION = """
     mutation CreatePage(
-            $slug: String, $title: String, $content: String, $pageType: ID!
-            $contentJson: JSONString, $isPublished: Boolean,
+            $slug: String, $title: String, $content: JSONString, $pageType: ID!
+            $isPublished: Boolean,
```

Comment:
Shouldn't we leave tests with both `contentJson` and `content` as we didn't drop it, but only deprecated it? The same situation in the other tests.

Reply:
We deprecate it in types but remove it in mutations (mutations are internal API and we will fix the dashboard accordingly).